test/index.test.ts

```diff
@@ -1 +1,71 @@
-test('returns the correct checksum for a file', async () => {});
+import { extractFileMetadata } from "../src/index"
+import * as fs from "fs"
+
+test("returns the correct checksum for a file", async () => {
+  const blob = new Blob([new Uint8Array(fs.readFileSync("test/testfile"))])
+
+  const result = await extractFileMetadata([new File([blob], "")])
+  expect(result[0].checksum).toEqual(
+    "e2d0fe1585a63ec6009c8016ff8dda8b17719a637405a4e23c0ff81339148249"
```

Comment:
It would be good to test to have a test that this checksum is independent of the chunk size, to catch issues with chunking.

Reply:
Yeah, that's true, I'll add one in.